src/shared/components/mutationTable/column/MutationAssessorColumnFormatter.tsx

```diff
@@ -260,7 +260,9 @@ export default class MutationAssessorColumnFormatter
         );
 
         // add tooltip for valid values
-        if (text.length > 0 && text !== NA)
+        if (text.length > 0 &&
+            text.toLowerCase() !== "na" &&
+            text.toLowerCase().indexOf("not available") === -1)
```

Comment:
Can we have one utility function for checking NA values in a string `isNA()` or something?